src/driver/sap/SapQueryRunner.ts

```diff
@@ -190,7 +192,7 @@ export class SapQueryRunner extends BaseQueryRunner implements QueryRunner {
                const queryStartTime = +new Date();
                const isInsertQuery = query.substr(0, 11) === "INSERT INTO";
 
-               const statement = databaseConnection.prepare(query);
+               statement = databaseConnection.prepare(query);
                statement.exec(parameters, (err: any, result: any) => {
 
                    // log slow queries if maxQueryExecution time is set
```

Comment:
I tested it with drop here and it worked

Reply:
`drop` is async as well?  Ouch..